### PR TITLE
Add compass and compass runtime agent installer CRs

### PR DESCRIPTION
**Pull request by tgorgol**
**Description**

Changes proposed in this pull request:

- Adds two new installer CRs:
  -  `installer-cr-cluster-compass.yaml.tpl` - `compass` component with just the required kyma components
  -  `installer-cr-cluster-runtime.yaml.tpl` - full kyma installation with included `compass-runtime-agent` component
- Modifies script generating artifacts to also generate installer artifacts for new CRs
- Adds README describing Installer CRs

**Related issue(s)**

#5243

### CODE REVIEW

File: installation/resources/installer-cr-cluster-agent.yaml.tpl [hunk 1]
```diff
@@ -0,0 +1,81 @@
+apiVersion: "installer.kyma-project.io/v1alpha1"
```
Comment (pkosiec) [inline installation/resources/installer-cr-cluster-agent.yaml.tpl]: Please add document describing which files should be modified when a new component is added.

File: installation/resources/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+|----------------|------|------|
+| `installer-cr.yaml.tpl` | Complete local Kyma installation | ✅ |
+| `installer-cr-cluster.yaml.tpl` | Complete cluster Kyma installation | ✅ |
+| `installer-cr-cluster-with-compass.yaml.tpl` | Complete cluster Kyma installation with `compass` and `compass-runtime-agent` components | ⛔️ |
+| `installer-cr-cluster-compass-minimal.yaml.tpl` | Cluster Compass installation only with Kyma components used by Compass | ⛔️ |
+| `installer-cr-cluster-agent.yaml.tpl` | Complete cluster Kyma installation with `compass-runtime-agent` component | ⛔️ |
+
+### Adding new components to Kyma
+
+When adding new components to Kyma remember to update every Installer CR that should use the new component.
```
Comment (pkosiec) [inline installation/resources/README.md]: Convert to NOTE and put it below Overview

File: installation/resources/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+## Overview
+
+This directory contains resources used by installer.
+
+## Installer CRs
+
+Files starting with `installer-cr-*` contain different component configurations that can be provided to installer. Currently available configurations:
+
+| Configuration | Description | Officially supported |
```
Comment (pkosiec) [inline installation/resources/README.md]: Add a new headline "Available configurations"

File: installation/resources/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Installer resources
+
+## Overview
+
+This directory contains resources used by installer.
+
+## Installer CRs
+
+Files starting with `installer-cr-*` contain different component configurations that can be provided to installer. Currently available configurations:
```
Comment (pkosiec) [inline installation/resources/README.md]: Remove "Currently available configurations:"

File: installation/resources/README.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+| Configuration | Description | Officially supported |
+|----------------|------|------|
+| `installer-cr.yaml.tpl` | Complete local Kyma installation | ✅ |
+| `installer-cr-cluster.yaml.tpl` | Complete cluster Kyma installation | ✅ |
```
Comment (pkosiec) [inline installation/resources/README.md]: Maybe a bullet point list would be better, with compass jobs marked as `(Experimental)`

File: installation/resources/README.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# Installer resources
+
+## Overview
+
+This directory contains resources used by installer.
```
Comment (klaudiagrz) [inline installation/resources/README.md]:
```suggestion
This directory contains resources used by the Installer.
```

File: installation/resources/README.md (hunk 4 above)
Comment (klaudiagrz) [inline installation/resources/README.md]:
```suggestion
Files starting with `installer-cr-*` contain different component configurations that you can provide to the Installer. These are the currently available configurations:
```

File: installation/resources/README.md (hunk 3 above)
Comment (klaudiagrz) [inline installation/resources/README.md]:
```suggestion
| Configuration | Description | Supported |
```

File: installation/resources/README.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+## Installer CRs
+
+Files starting with `installer-cr-*` contain different component configurations that can be provided to installer. Currently available configurations:
+
+| Configuration | Description | Officially supported |
+|----------------|------|------|
+| `installer-cr.yaml.tpl` | Complete local Kyma installation | ✅ |
```
Comment (klaudiagrz) [inline installation/resources/README.md]:
```suggestion
| `installer-cr.yaml.tpl` | Provides complete local Kyma installation. | ✅ |
```

File: installation/resources/README.md (hunk 5 above)
Comment (klaudiagrz) [inline installation/resources/README.md]:
```suggestion
| `installer-cr-cluster.yaml.tpl` | Provides complete cluster Kyma installation. | ✅ |
```

File: installation/resources/README.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+| `installer-cr-cluster.yaml.tpl` | Complete cluster Kyma installation | ✅ |
+| `installer-cr-cluster-with-compass.yaml.tpl` | Complete cluster Kyma installation with `compass` and `compass-runtime-agent` components | ⛔️ |
```
Comment (klaudiagrz) [inline installation/resources/README.md]:
```suggestion
| `installer-cr-cluster-with-compass.yaml.tpl` | Provides complete cluster Kyma installation with the `compass` and `compass-runtime-agent` components. | ⛔️ |
```

File: installation/resources/README.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+| `installer-cr-cluster-compass-minimal.yaml.tpl` | Cluster Compass installation only with Kyma components used by Compass | ⛔️ |
```
Comment (klaudiagrz) [inline installation/resources/README.md]:
```suggestion
| `installer-cr-cluster-compass-minimal.yaml.tpl` | Provides cluster Compass installation only with the Kyma components that Compass uses. | ⛔️ |
```

File: installation/resources/README.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+| `installer-cr-cluster-agent.yaml.tpl` | Complete cluster Kyma installation with `compass-runtime-agent` component | ⛔️ |
```
Comment (klaudiagrz) [inline installation/resources/README.md]:
```suggestion
| `installer-cr-cluster-agent.yaml.tpl` | Provides complete cluster Kyma installation with the `compass-runtime-agent` component. | ⛔️ |
```

File: installation/resources/README.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+
+### Adding new components to Kyma
```
Comment (klaudiagrz) [inline installation/resources/README.md]:
```suggestion
## Add a new component to Kyma
```

File: installation/resources/README.md (hunk 2 above)
Comment (klaudiagrz) [inline installation/resources/README.md]:
```suggestion
When you add a new component to Kyma, you must update every Installer CR that will use the component.
```

File: installation/resources/README.md [hunk 12]
```diff
@@ -0,0 +1,22 @@
...
+
+This directory contains resources used by the Installer.
+
+>**NOTE:** When you add a new component to Kyma, you must update every Installer CR that will use the component.
+
+## Installer CRs
+
+Files starting with `installer-cr-*` contain different component configurations that you can provide to the Installer.
+
+## Available configurations
```
Comment (klaudiagrz) [inline installation/resources/README.md]:
```suggestion
### Available configurations
```

File: installation/resources/README.md [hunk 13]
```diff
@@ -0,0 +1,22 @@
...
+
+| Configuration | Description | Part of release |
```
Comment (klaudiagrz) [inline installation/resources/README.md]:
```suggestion
| Configuration | Description | Part of the release |
```

Comment (klaudiagrz) [reply installation/resources/README.md]: Please add a sentence that will separate the heading and the table.

File: installation/resources/README.md [hunk 14]
```diff
@@ -0,0 +1,23 @@
...
+
+## Installer CRs
+
+Files starting with `installer-cr-*` contain different component configurations that you can provide to the Installer.
+
+>**NOTE:** When you add a new component to Kyma, you must update every Installer CR that will use the component.
+
+### Available configurations
+
+These are the currently available configurations:
```
Comment (klaudiagrz) [inline installation/resources/README.md]:
```suggestion
These are the currently available Installer configurations:
```

**Pull request merged**